### ISSUE TITLE
Set up DataSync between Staging EFS and Staging Public S3

### ISSUE THREAD

**Issue by timcosgrove** [department-of-veterans-affairs/va.gov-cms#16928]
## Requirements

We need files to sync from Staging's EFS to the Public S3 bucket for Stagin, so that files added via the Drupal file system are available publicly. 

```[tasklist]
### Acceptance criteria
- [ ] A Datasync process is set up between Staging EFS and Staging Public S3
- [ ] Once all files are sync'd, file sync tests are run to learn how long sync of new files should take
```

## Background & implementation details

EFS: https://us-gov-west-1.console.amazonaws-us-gov.com/efs/home?region=us-gov-west-1#/file-systems/fs-3b9aa63a
S3: https://us-gov-west-1.console.amazonaws-us-gov.com/s3/buckets/dsva-vagov-staging-cms-files?region=us-gov-west-1&tab=objects

EFS path: all files under docroot/sites/default/files/*
S3 destination path: img/*

Once the sync is set up and the initial sync is complete, we should add files to the Staging CMS and track how long they take to end up on S3. Drupal folks on the team can help with identifying where the files will end up on Drupal so we can observe on the S3 end.

**Comment by timcosgrove** [comment]
closing in favor of #16929
